### PR TITLE
Fixed issue #1 - build multi-arch images

### PR DESCRIPTION
**Pull request by rbo**
@juliaaano thanks for the hint to the docs, works like a charm:

```
% skopeo inspect --raw docker://ghcr.io/rbo/antora-viewer:main | jq
{
  "schemaVersion": 2,
  "manifests": [
    {
      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
      "digest": "sha256:02b992371294352ec09aeebc2b9bd4728d667490c7661fc848d178f8417acdac",
      "size": 926,
      "platform": {
        "architecture": "amd64",
        "os": "linux"
      }
    },
    {
      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
      "digest": "sha256:6289a6c2e39cd447fe2a0312ad9e0147bc525e19071067dcd3d1007e22631fc9",
      "size": 926,
      "platform": {
        "architecture": "arm64",
        "os": "linux"
      }
    }
  ]
}
```

### CODE REVIEW

**Comment by rbo** [comment]
Fix for issue #1

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -37,18 +37,17 @@ jobs:
       - name: set datetime
         run: |
           echo "datetime=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
-      - name: log in registry.redhat.io
```
Comment (juliaaano) [inline .github/workflows/cicd.yml:40]: I believe this should not be removed as it is required for https://github.com/juliaaano/antora-viewer/blob/e80903cd5015dfb072c42d1b0dfc051190d68381/container/Containerfile#L1

**Review by juliaaano — CHANGES_REQUESTED** [review]
I'm so grateful for your PR. Thank you for taking time.

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (rbo) [reply .github/workflows/cicd.yml:40]: No, registry.redhat.io need auth. 

registry.access.redhat.com works without auth and provide only "free" available images like the ubi

Comment (juliaaano) [reply .github/workflows/cicd.yml:40]: This is great. I guess I'll review a bunch of other actions where I'm doing that unecessarly. Thanks!

**Pull request merged**